src/app.module.ts

```diff
@@ -11,13 +11,9 @@ import configuration from './global/config/configuration';
 import { RatesModule } from './rates/rates.module';
 import { LoggerModule } from './global/logger/logger.module';
 import { NotifierModule } from './global/notifier/notifier.module';
+import { Logger } from './global/logger/logger.service';
 
-const {
-  MONGODB_HOST = '127.0.0.1',
-  MONGODB_PORT = 27017,
-  REDIS_HOST = 'localhost',
-  REDIS_PORT = 6379,
-} = process.env;
+const MONGODB_NAME = 'tickersdb';
```

Comment:
1. Is the DB structure compatible with old currencyinfo service? Both are using the [same](https://github.com/Adamant-im/adamant-currencyinfo-services/blob/master/db/mongodb.js#L13) DB name.

2. Maybe rename to "currencyinfo" and move that option to `config.jsonc`